lib/OpenQA/WebAPI.pm

```diff
@@ -130,6 +130,7 @@ sub startup {
     $test_r->get('/')->name('test')->to('test#show');
     $test_r->get('/ajax')->name('job_next_previous_ajax')->to('test#job_next_previous_ajax');
     $test_r->get('/modules/:moduleid/fails')->name('test_module_fails')->to('test#module_fails');
+    $test_r->get('dependencies')->name('test_dependencies')->to('test#dependencies');
```

Comment:
for consistency either have all $test_r->get calls with / or none.

Reply:
fixed